app/views/npq_separation/admin/courses/show.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= govuk_back_link(href: url_for(:back)) %>
+
+<h1 class="govuk-heading-l"><%= @course.name %></h1>
+
+<%=
+  govuk_summary_list do |sl|
+    sl.with_row do |row|
+      row.with_key(text: "ID")
+      row.with_value(text: @course.id)
+    end
+
+    sl.with_row do |row|
+      row.with_key(text: "ECF ID")
+      row.with_value(text: @course.ecf_id)
+    end
+
+    sl.with_row do |row|
+      row.with_key(text: "Position")
+      row.with_value(text: @course.position)
+    end
+
+    sl.with_row do |row|
+      row.with_key(text: "Name")
+      row.with_value(text: @course.name)
+    end
+
+    sl.with_row do |row|
+      row.with_key(text: "Description")
+      row.with_value(text: @course.description)
+    end
+
+    sl.with_row do |row|
+      row.with_key(text: "Display")
+      row.with_value(text: boolean_red_green_tag(@course.display?))
+    end
```

Comment:
Was going to suggest we put the course identifiers in here too but [they're all hardcoded](https://github.com/DFE-Digital/npq-registration/blob/main/app/models/course.rb) 🙈 

I think in collaboration with @DFE-Digital/npq-cpd we need to move these into the model.